Makefile

```diff
@@ -61,7 +61,7 @@ coverage:
 	hack/dockerized "./hack/coverage.sh ${WHAT}"
 
 goveralls: go-build
-	SYNC_OUT=false hack/dockerized "COVERALLS_TOKEN_FILE=${COVERALLS_TOKEN_FILE} COVERALLS_TOKEN=${COVERALLS_TOKEN} CI_NAME=prow CI_BUILD_NUMBER=${BUILD_ID} CI_BRANCH=${PULL_BASE_REF} CI_PULL_REQUEST=${PULL_NUMBER} ./hack/goveralls.sh"
+	SYNC_OUT=false hack/dockerized "COVERALLS_TOKEN_FILE=${COVERALLS_TOKEN_FILE} COVERALLS_TOKEN=${COVERALLS_TOKEN} CI_NAME=prow COVERALLS_SERVICE_JOB_ID=${BUILD_ID} CI_BRANCH=${PULL_BASE_REF} CI_PULL_REQUEST=${PULL_NUMBER} ./hack/goveralls.sh"
```

Comment:
Not sure why that works as the PULL_BASE_REF points to `master`? Normally you should use PULL_REFS which is what I am doing in my PR?

Reply:
Not sure... But I took reference from [here](https://docs.coveralls.io/api-reference): `You must specify either repo_token or a service name and job id.` These are required fields and others are only optional. 

`service_pull_request` is used for updating the status.

Reply:
Actually we _are_ providing the repo token (by using COVERALLS_TOKEN_FILE env var instead of providing it via the command line option.